helm/terraform-job/templates/terraform-apply.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: terraform-apply
+  labels:
+    component: infrastructure
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": pre-install, pre-upgrade
```

Comment:
`helm.sh/hook-weight: "10"`

Reply:
Made sure all the templates have the hook weight ordering with 49bfe48.